### PR TITLE
[ORDERS] Implements random starting areas for the CEV Eris. One of 3 default l…

### PR DESCRIPTION
**Pull request by KirkwallDay**
…ocations will start where the CEV Eris is. It's random each round, and more can be added.

## About The Pull Request
This request fills an order from the Discord to have the Eris start at one of the three areas on the overmap.
Capture

The Three locations are The asteroid, Abandonded Fortress, or Ruins. This is accomplished in a somewhat strange way, since the Eris is spawned first, I had to make it so the random landmark will spawn at the Eris instead. This also means the asteroid loses it's default spawn status at the Eris, and its default coordinates are now randomized, just like the fortress and ruins are.

It's also built to be expandable, create more sectors, and then add them to the list.

## Why It's Good For The Game

It should make rounds more varied, and offer players a chance to see the other areas, even when there's no captain or anyone driving the ship.

## Changelog
:cl: yorpyorpyorp
add: Eris will now randomly start at 1 of 3 locations.
add: Lines in config file to determine starting location.
del: Asteroid loses its default location.
/:cl:

### CODE REVIEW

File: code/modules/overmap/sectors.dm
Comment (seekin4u) [inline code/modules/overmap/sectors.dm:20]: init it to false, not to `NULL`

Comment (KirkwallDay) [reply code/modules/overmap/sectors.dm]: Good point, change made!

**Review by genessee596 — APPROVED** [review]
This PR gets the Delegate stamp of approval.

It'd be nice to see other ruins more than once in a blue moon. This might also encourage people to make more ruins as well, which is nice.

Still needs actual code review from a maintainer.

**Review by TorinoFermic — APPROVED** [review]

File: code/modules/overmap/sectors.dm
Comment (TheShown911) [inline code/modules/overmap/sectors.dm:41]: The whole way the ship gets found is kinda strange, but hopefuly random start will always stay on so it shouldn't be a big problem.

Comment (TorinoFermic) [reply code/modules/overmap/sectors.dm:41]: I hope our code has this eris ship being already in a global var/list

Comment (KirkwallDay) [reply code/modules/overmap/sectors.dm]: I can't find it in the global vars, so I made a new var list for ships and put it in there.
I added configuration file support. 
Now that the eris no longer needs to be 'found' that part is gone.

**Comment by genessee596** [comment]
This has gotten approval from maintainers and delegates, so I'm going to go ahead and merge this.

**Pull request merged**
